test/unit/dynamic-bootstrap-specs.js

```diff
@@ -44,7 +44,7 @@ describe('dynamic bootstrap', function () {
     rimraf(process.env.APPIUM_BOOTSTRAP_DIR)
 
       // first call: should create new bootstrap file
-      .then(function () { return prepareBootstrap(); })
+      .then(async function () { return await prepareBootstrap(); })
```

Comment:
This `await` is not necessary (and without it, the `async` can go away). In this sequence of calls, returning a promise inside a `then` resolver is fine and will work out. Same with the other places below.